src/features/vault/components/PoolSummary/PoolSummary.js

```diff
@@ -34,7 +34,13 @@ const PoolSummary = ({
   const vaultStateTitle = useMemo(() => {
     let state =
       pool.status === 'eol'
-        ? t('Vault-DepositsRetiredTitle')
+        ? pool.retireReason === 'rewards'
+          ? t('Vault-DepositsRetiredRewardsTitle')
+          : pool.retireReason === 'exploit'
+          ? t('Vault-DepositsRetiredExploitTitle')
+          : pool.retireReason === 'tvl'
+          ? t('Vault-DepositsRetiredTVLTitle')
+          : t('Vault-DepositsRetiredTitle')
```

Comment:
Can you make this a map (object) for better readability. You would also be able to move the object to its own file.

I'm thinking replacing this with something like:

```ts
const retireReasonMap = {
  rewards: "Vault-DepositsRetiredRewardsTitle",
  exploit: "Vault-DepositsRetiredExploitTitle",
  // ...
}

const getRetireReason = (key) => {
  if (key in retireReasonMap) {
    const reason = retireReasonMap[key];
    return reason
  } else {
   return ""
  }
}
```

Reply:
I've implemented your suggestion, please have a look at the latest commit.

Reply:
A maintainer will take a look at it and merge it if it looks good (I'm not a maintainer on this repo)